src/components/footer/footer.js

```diff
@@ -1,8 +1,64 @@
 import React from "react";
 import styles from "./footer.module.scss";
-// import twitterIcon from "./images/twitter-icon.png"
+import dfeLogo from "../../pages/home-page/images/dfeLogo.png";
+import fbLogo from "../../pages/home-page/images/fbLogo.png";
+import instaLogo from "../../pages/home-page/images/instaLogo.png";
+import linkedinLogo from "../../pages/home-page/images/linkedinLogo.png";
+import twitterLogo from "../../pages/home-page/images/twitterLogo.png";
```

Comment:
Can you move these to an images folder inside the footer folder and name the actual filenames with hyphens instead of camelcase